docs/process/functions.md

```
@@ -0,0 +1,68 @@
+# PetriNet functions
+
+This document contains information about new Petriflow functions introduced in NAE 5.6.0.
+
+## Overview
+
+Petriflow functions provide opportunity to declare your own functions without any changes in applications code. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)